test/manual/js/index.js

```diff
@@ -2,13 +2,20 @@
  * Copyright (c) NAVER Corp.
  */
 var log = document.getElementById("log");
+var isStop = false;
 function addLog(msg) {
     log.innerHTML = msg + "\n" + log.innerHTML;
 }
 document.getElementById("clear").addEventListener("click", function(e) {
     log.innerHTML = "";
 });
 
+document.getElementById("stop").addEventListener("click", function(e) {
+	setTimeout(function () {
+		isStop = true;
+	}, 1000);
```

Comment:
@younkue What does duration(`1000`) mean?

Reply:
@jongmoon  This is a good time to test the stop manual. It doesn't mean anything.